frontend/src/modules/entitydetails/components/EntityNavigation.js

```diff
@@ -0,0 +1,66 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import './EntityNavigation.css';
+
+import type { DbEntity } from 'modules/entitieslist';
+
+
+type Props = {|
+    +entity: DbEntity,
+|};
+
+
+/**
+ * Component showing entity navigation toolbar.
+ *
+ * Shows next/previous buttons.
+ */
+export default class EntityNavigation extends React.Component<Props> {
+    goToNextEntity = (entity: DbEntity) => {
+        console.log("foo");
+    }
+
+    goToPreviousEntity = (entity: DbEntity) => {
+        console.log("bar");
+    }
+
+    render(): React.Node {
+        const { entity } = this.props;
+
+        return <div className='entity-navigation clearfix'>
+            <Localized
+                id="entitynavigation-next"
+                attrs={{ title: true }}
+                glyph={
+                    <i className="fa fa-chevron-down fa-lg"></i>
+                }
+            >
+                <button
+                    id="next"
```

Comment:
I like to avoid using ids on React components, especially ones that are as generic as this one. The idea being that, components being reusable, you run the risk of creating conflicts in the future. Thus far I've removed all (I believe) IDs and used on classes. CSS selectors are good enough that we can do that and still not create styling conflicts.